stakingModel/contracts/InterestDistribution.sol

```diff
@@ -156,12 +162,12 @@ library InterestDistribution {
       * AccumulatedYieldPerToken = AccumulatedYieldPerToken(P) + (Daily Interest/GrandTotalStaked)
       * 
       * @param _dailyIntrest            Newly accrued interest (Precision same as G$ = 2)
-      * @param _grandTotalStaked        Total staked by all stakers. (Precision points = 18)
+      * @param _globalTotalStaked       Total staked by all stakers. (Precision points = 18)
       * 
       * @return  new yield since last update with same precision points as G$(2).
     */
-    function getAccumulatedYieldPerToken(uint256 _dailyIntrest, uint256 _grandTotalStaked) internal view returns(uint256) {
-      return _dailyIntrest.mul(DECIMAL1e18).div(_grandTotalStaked);
+    function getGlobalYieldPerToken(uint256 _dailyIntrest, uint256 _globalTotalStaked) internal view returns(uint256) {
+      return _dailyIntrest.mul(DECIMAL1e18).div(_globalTotalStaked);
```

Comment:
this is just the daily yield either rename this method or make it return
currentGlobalYield + dailyYield

Reply:
i suggest to do getCurrentGlobalYieldPerToken(interestData, dailyinterest)
return current+dialy